### DISCUSSION TITLE
[5.0] Table view for matrix fields

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#14014] [Ideas]
As of the latest alpha, Craft 5 offers 3 view modes for matrix fields:
- Card
- Inline
- Element index (cards/table)

In the same vein as the `Card` view (like a simplifed element card index), I would like to propose a fourth option: `Table view`.

This would be similar to the table inside a table index view, but without the surrounding "chrome" (search, filters and such). 

I think this would be a nice alternative to the standard table field, when we need the power of matrix in a compact UI. It could also mean that the planned addition of relational column types in table fields would not be so important, or even necessary (though I would still very much like to see that if it would provide a "lighter" alternative to using matrix).

**Comment by alexanderbuergin** [comment]
I would love to see that.

**Comment by AugustMiller** [comment]
I acknowledge this isn't 100% what you're describing, but I thought it was worth pointing out!

Element indexes (both the main “sources” view and when used as the view mode for Matrix fields) have an **Edit** button that allows editing multiple elements at once:

![Screen Shot 2024-01-30 at 10 27 04](https://github.com/craftcms/cms/assets/1895522/f57f9c39-c555-4256-b1dc-05563175e232)

**Reply by thupsi** [reply]
I'm aware of that and indeed the edit functionality is awesome! What I'm describing is basically this view, without the element index chrome. We have 'card view' in an element index AND vanilla 'card view', so I'm ALSO proposing vanilla 'table view'. Hope that makes sense 😁
